torchgeo/trainers/detection.py

```diff
@@ -238,8 +238,12 @@ def training_step(
         """
         x = batch['image']
         batch_size = x.shape[0]
+        # Get bbox key as it can be one of {"bbox", "bbox_xyxy", "bbox_xywh"}
```

Comment:
Are our models compatible with all 3, or do we need to assert that a specific format is used?